.github/workflows/build-test-deploy.yaml

```diff
@@ -19,10 +19,11 @@ concurrency:
 
 jobs:
   call-workflow:
-    uses: dhis2-sre/gha-workflows/.github/workflows/im-build-test-deploy.yaml@master
+    uses: dhis2-sre/gha-workflows/.github/workflows/im-build-test-deploy.yaml@healthcheck
```

Comment:
Just a reminder to update this before merging

Reply:
thank you!